### PR TITLE
Fix builds for 1.13

### PR DESCRIPTION
**Pull request by mpysiak**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -21,11 +21,10 @@ jobs:
...
             matrix:
-                php: ["8.0", "8.1"]
-                symfony: ["5.4.*", "^6.0"]
-                sylius: ["^1.12"]
-                node: ["14.x", "16.x", "18.x"]
-                mysql: ["5.7", "8.0"]
+                php: ["8.1", "8.2", "8.3"]
+                symfony: ["^5.4", "^6.4"]
+                node: ["20.x"]
+                mysql: ["8.3"]
```
Comment (GSadee) [inline .github/workflows/build.yml]:
```suggestion
                mysql: ["8.0"]
```
I would be for testing the LTS version, in the near future, we could probably add 8.4 as the newly released LTS soon

File: composer.json [hunk 2]
```diff
@@ -8,14 +8,15 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "^8.0",
-        "sylius/sylius": "^1.12",
-        "sylius/mailer-bundle":  "^1.8 || ^2.0@beta",
+        "php": "^8.1",
+        "sylius/sylius": "~1.13.0",
+        "symfony/flex": "^2.4",
```
Comment (GSadee) [inline composer.json]: Why did you move flex from dev dependencies? I believe it makes more sense there 🤔

File: tests/Application/config/bundles.php [hunk 3]
```diff
@@ -58,4 +58,5 @@
     Sylius\Calendar\SyliusCalendarBundle::class => ['all' => true],
     Symfony\WebpackEncoreBundle\WebpackEncoreBundle::class => ['all' => true],
     League\FlysystemBundle\FlysystemBundle::class => ['all' => true],
+    Sylius\Abstraction\StateMachine\SyliusStateMachineAbstractionBundle::class => ['all' => true],
```
Comment (GSadee) [inline tests/Application/config/bundles.php]: It could be probably added before other Sylius bundles

**Review by GSadee — APPROVED** [review]

**Review by Wojdylak — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, Michał! :1st_place_medal:
